app/views/layouts/application.html.haml

```diff
@@ -12,3 +12,6 @@
         = yield
 
     = render 'shared/footer'
+
+  :javascript
+  var navigation = $('#nav-main').okayNav();
```

Comment:
You have not indented this. The plugin is not even initializing. Also uncomment the CSS.